rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -60,10 +61,9 @@ std::mutex g_savedata_mutex;
 
 static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version, vm::cptr<char> dirName,
 	u32 errDialog, PSetList setList, PSetBuf setBuf, PFuncList funcList, PFuncFixed funcFixed, PFuncStat funcStat,
-	PFuncFile funcFile, u32 container, u32 unknown, vm::ptr<void> userdata, u32 userId, PFuncDone funcDone)
+	PFuncFile funcFile, u32 container, u32 unknown, vm::ptr<void> userdata, u32 userId)
```

Comment:
Nope, this megafunction was modelled after an existing one. The arguments, their positions were preserved.